### ISSUE TITLE
moinwiki - transclusions within tables fails when alt text is null

### ISSUE THREAD

**Issue by RogerHaase** [moinwiki/moin#1315]
When sample data is loaded this fails:

    ||{{jpeg||width=160}}||

This works:

    ||{{jpeg|kitty|width=160}}||

    ||{{jpeg| |width=120}}||

Related: #386

**Comment by RogerHaase** [comment]
More complex examples should also work:

    ||zzz{{jpeg||width=160}}zzz{{jpeg||width=160}}zzz||zzz{{jpeg||width=160}}zzz{{jpeg||width=160}}zzz||a||b||
